src/kOS.Safe/Encapsulation/ConstantValue.cs

```diff
@@ -9,8 +9,20 @@ static ConstantValue()
         {
             AddGlobalSuffix<ConstantValue>("G", new StaticSuffix<double>(() => 6.67384*Math.Pow(10,-11)));
             AddGlobalSuffix<ConstantValue>("E", new StaticSuffix<double>(() => Math.E));
-            AddGlobalSuffix<ConstantValue>("e", new StaticSuffix<double>(() => Math.E));
             AddGlobalSuffix<ConstantValue>("PI", new StaticSuffix<double>(() => Math.PI));
+            AddGlobalSuffix<ConstantValue>("C", new StaticSuffix<double>(() => 299792458.0)); // spd of light in m/s.
```

Comment:
See:
doc/source/math/basic.rst, lines 68 - 78.

It's still used if you have RT installed.